reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -497,9 +497,27 @@ def expand_1d_to_3d(arr_1d):
         arr_3d : arr
             Shape (spherical coords)
         """
-        n_pix = arr_1d.shape[0] * 2
-        arr_3d = np.ones((n_pix, n_pix, n_pix))
-        # arr_1d fsc_1d to 3d (spherical shells)
+        half_pix = arr_1d.shape[0]
+        corner_111 = np.zeros((half_pix, half_pix, half_pix))
+
+        x = np.linspace(0, half_pix, half_pix)
```

Comment:
See my comment https://github.com/compSPI/reconstructSPI/issues/20#issuecomment-1082042764

You probably want to use np.arange not np.linspace to get the right coordinates. np.arange(-n//2,n//2) would give [-n//2, ..., -n//2-1] (good) instead of having the two end points be -n//2,n//2 in n steps, so that there would be non integer values (e.g. counting from -3 to +3 in 6 steps when you have to also count zero, so there's seven numbers between them).